Sources/Internal/FrameProvider.swift

```diff
@@ -288,14 +283,23 @@ final class FrameProvider {
     }
   }
 
+  private func minXOfItem(
+    at dayOfWeekPosition: DayOfWeekPosition,
+    minXOfContainingRow: CGFloat)
+    -> CGFloat
+  {
+    let distanceFromMonthLeadingEdge = CGFloat(dayOfWeekPosition.rawValue - 1) *
+      (daySize.width + content.horizontalDayMargin)
+    return minXOfContainingRow + content.monthDayInsets.left + distanceFromMonthLeadingEdge
+  }
+
```

Comment:
note: the changes in this file should be a NOP. I noticed some duplicated math and wanted to consolidate some of the calculations where it made sense. These changes in the `FrameProvider` have 0 effect on the RTL support I added, and I confirmed they don't break anything since all `FrameProvider` tests pass.